### DISCUSSION TITLE
Is there a simple way to read all values in a column?

### DISCUSSION THREAD

**Discussion by timothy-mccarthyfinch** [exceljs/exceljs#2488] [Q&A]
I would like values by column, is there an efficient way to do this?

regards.

**Comment by MusicDev33** [comment]
https://github.com/exceljs/exceljs#columns I'd start here, specifically this line:
```
dobCol.eachCell(function(cell, rowNumber) {
  // ...
});
```

This is assuming you already know how to read the file into a workbook, then get the worksheet. The overall implementation would look like this:
```
const dobCol = worksheet.getColumn(3);

dobCol.eachCell(function(cell, rowNumber) {
  // ...
});
```
